modules/javafx.graphics/src/main/java/com/sun/prism/sw/SWGraphics.java

```diff
@@ -661,6 +661,7 @@ public void drawString(GlyphList gl, FontStrike strike, float x, float y,
                 (strike.getAAMode() == FontResource.AA_LCD) &&
                 getRenderTarget().isOpaque() &&
                 (this.paint.getType() == Paint.Type.COLOR) &&
+                ((Color)this.paint).getAlpha() == 1.0f &&
```

Comment:
Could avoid the cast here if you want:

```suggestion
                this.paint instanceof Color c && 
                c.getAlpha() == 1.0f &&
```

Reply:
I had thought of it earlier, but didn't do it at the time. I made this change along with the suggested test changes. Thanks for the suggestions.